CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Fixed
+- Unsafe matching of comp rep columns in `get_comp_rep_parameter_indices`
```

Comment:
Either `comp_rep` or "computational representation", but this is _neither fish nor meat_.

Reply:
agree, fixed, ty